src/models/thelio-mira-b4-n3/repairs.md

```diff
@@ -0,0 +1,613 @@
+
+### Steps to install the thermal paste/CPU cooler:
+
+1. Draw an `X` shape of thermal paste onto the CPU.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+2. Place the CPU cooler onto the CPU; while holding it in place, screw its vertical mounting bar into the horizontal mounting brackets.
+    - The cooler should be oriented so the Thermalright logos are on the bottom.
+    - Insert each screw partially, then finish tightening both.
+3. Set the computer upright, then reinstall the cooler-mounted CPU fan, CPU duct, GPU, side brace, and top case.
+
+## Replacing the power supply:
+
+The power supply unit (PSU) can be replaced with another unit of the same model.
+
+**Part numbers:**
+- The default PSU in units shipped after July 2024 is the [XPG Core Reactor II](https://www.xpg.com/us/xpg/pc-components-core-reactor-ii).
+- The default PSU in older units is the [Be Quiet! Pure Power 12 M](https://www.bequiet.com/en/powersupply/4162).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 45 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the power supply:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), and [unplug the GPU (if installed)](#replacing-the-gpu).
+2. Unscrew the power supply bracket's left side from the bottom of the chassis.
+
+![Power supply bracket screw (left)](./img/psu-bracket-screw-left.webp)
+
+3. Unscrew and remove the cable management bracket on the right side of the chassis, then unscrew the power supply's right side from the chassis.
+
+![Cable management bracket](./img/psu-cable-management-bracket.webp)
+![Power supply bracket screw (right)](./img/psu-bracket-screw-right.webp)
+
+4. Unscrew the power supply's four screws from the back of the chassis.
+
+![Power supply screws](./img/psu-screws.webp)
+
+5. Unplug all of the modular cabling from the back of the PSU.
+
+![PSU cabling](./img/psu-cables.webp)
+
+6. Remove/replace the power supply.
+    - The replacement PSU should be installed with the fan facing the bottom of the case.
+    - Place the PSU on the rubber post and hold it against the screwholes in the back of the case while attaching it.
+7. Connect the power cables to the new power supply and GPU (if installed), then reinstall the PSU brace, side brace, and top case.
+
+## Replacing the Thelio Io board:
+
+Named after Jupiter's moon Io, the Thelio Io daughterboard handles the front power button and fan control. If the Thelio Io board becomes defective, it can be replaced using the instructions below.
+
+**Part numbers:**
+- Thelio Mira B4-N3 uses [Thelio Io version 2](https://github.com/system76/thelio-io/tree/92c2a60b6d6acc0f9d51d819cba07ad28e2b03f3) ([PCB revision thelio_io_2.3](https://github.com/system76/thelio-io-hardware/tree/91814370002f762ac01cd1dd99dcbb43336352ab/pcb-thelio-io).
```

Comment:
Looks like this line is missing a closing parentheses `)` for the PCB revision note.